spec/models/placements/provider_spec.rb

```diff
@@ -46,6 +46,25 @@
         expect(placements_provider.users).to all(be_a(Placements::User))
       end
     end
+
+    describe "#partnerships" do
+      it { is_expected.to have_many(:partnerships) }
+    end
+
+    describe "#parnter_schools" do
```

Comment:
```suggestion
    describe "#partner_schools" do
```

Reply:
resolved in [3602283](https://github.com/DFE-Digital/itt-mentor-services/pull/449/commits/3602283996a7608085d29cdac991157fd989e79d)